Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal.c

```diff
@@ -92,7 +92,7 @@
   * @{
   */
 __IO uint32_t uwTick;
-uint32_t uwTickPrio   = (1UL << __NVIC_PRIO_BITS); /* Invalid PRIO */
```

Comment:
__NVIC_PRIO_BITS is defined as 4 in the stm32f413xx.h file since this chipset uses 4 priority bits.